### PR TITLE
fixes #11 - Add Meme Generation

### LINKED ISSUE #11: Memegen Support
Would the maintainers be open to allowing a command, say
```
meme
```
in response to which DrinkWaterBot would respond with a link to a meme?

Future scope could include some content injection or meme templates, such as 
```
meme <template> <text of meme>
```
which would return a link to a meme generated through the [memegen API](https://api.memegen.link/).

I would like to work on this. Thank you :D

### PR DESCRIPTION
**Pull request by anirudhRowjee**
Bot friends can now generate memes from templates available at [this repository](https://github.com/jacebrowning/memegen), which is also the backend we're using to support this feature.

We aren't downloading the image ourselves, which should help the bot run better.

### CODE REVIEW

**Review by srujandeshpande — APPROVED** [review]
Looks great!

File: src/commands/meme.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    /*
+        * Usage: `meme <keyword> <top text (1 word)> <bottom text (1 word)>` to generate a new meme
+    */
+}
+
+
+
+function getMeme(message, template, top_text, bottom_text)
+{
+    console.log(message)
```
Comment (srujandeshpande) [inline src/commands/meme.js:18]: What are your thoughts on commenting out the logs? No other commands have logging, but would it be useful?

**Review by srujandeshpande — CHANGES_REQUESTED** [review]

File: src/commands/meme.js (hunk 1 above)
Comment (anirudhRowjee) [reply src/commands/meme.js:18]: Whoops, my bad. Remnants of the debugging phase. Will fix ASP!

Comment (anirudhRowjee) [reply src/commands/meme.js]: done!

**Review by srujandeshpande — APPROVED** [review]
lgtm

**Pull request merged**